Cargo.toml

```
@@ -15,7 +15,7 @@ env_logger = { version = "0.9", default-features = false }
 log = "0.4"
 object = { version = "0.26", default-features = false, features = ["read_core", "elf", "std"] }
 getrandom = "0.2"
-
+regex = "1.5"  
```

Please keep a new line to the next section.
```suggestion
regex = "1.5"

```